packages/squiggle-lang/src/library/registry/index.ts

```diff
@@ -53,8 +53,8 @@ const fnList: FRFunction[] = [
 export const registry = Registry.make(fnList);
 
 export const nonRegistryLambdas: [string, Lambda][] = [
-  ["mx", mxLambda],
-  ["mixture", mxLambda],
+  ["mx", mxLambda()],
+  ["mixture", mxLambda()],
```

Comment:
This code builds two different lambda objects which are identical.

A few thoughts:
- you could cache it in a local var and reuse
- but also this conversion from `mxLambda` to `mxLambda()` is a fix for cyclic imports which is not necessary (I checked)
- but also: actually, it might make sense to create two different lambdas, because lambdas carry their names which are sometimes exposed; if so, it would be `["mx", mxLambda("mx")], ["mixture", mxLambda("mixture")]`
  - I'm not sure if _builtin_ names are ever exposed, AFAICT they're currently used only for stacktraces, and you'd need an extra stack frame for that to happen (so `map` can show in the stacktrace, because it does calls its parameters, but `mx` usually won't); but that might change in the future